src/main/java/DukeException.java

```diff
@@ -0,0 +1,10 @@
+public class DukeException extends Exception {
+  public DukeException(String errorMsg) {
```

Comment:
I'm following the google style instead of the one mentioned in the textbook. Not sure if that is allowed
https://google.github.io/styleguide/javaguide.html#s4-formatting